### DISCUSSION TITLE
Next Release will have better Oculus Apps Viewer that will soon be used for profile management

### DISCUSSION THREAD

**Discussion by Eliminater74** [DevOculus-Meta-Quest/Oculus-VR-Dash-Manager#5] [Announcements]
![Screenshot 2023-11-08 175516](https://github.com/DevOculus-Meta-Quest/Oculus-VR-Dash-Manager/assets/8908541/2f7e6b8f-58fc-4861-b609-b55ad1b3b8ee)

**Comment by SirMonocoleGentlesir** [comment]
Wow that looks amazing
keep up the hard work
